lib/device/index.js

```diff
@@ -115,14 +120,12 @@ Device.prototype.setProperties = function() {
  * @returns {object}
  */
 Device.prototype.findConfiguration = function(deviceType, model) {
-  var key;
-
-  model = model.toUpperCase();
```

Comment:
Instead of re-writing the variable, let's add a new variable for the uppercase version. That way we aren't needlessly uppercasing the same value every time.